charts/generic-govuk-app/templates/service.yaml

```
@@ -20,4 +20,5 @@ spec:
       port: {{ .Values.service.port }}
       targetPort: {{ .Values.nginxPort }}
   selector:
-    app: {{ $fullName }}
+    # app: {{ $fullName }}
+    app: {{ .Values.service.selector.app | default ($fullName) }}
```

nit: the parens are superfluous.
```suggestion
    app: {{ .Values.service.selector.app | default $fullName }}
```